### DISCUSSION TITLE
Initializing (delayed)

### DISCUSSION THREAD

**Discussion by lukaszervudakis** [jomjol/AI-on-the-edge-device#2098] [Q&A] [answered]
Hi, I've just finished the setup and rebooted everything. However, for ~30 mins, the WebUI just shows the watermeter logo with "Initializing (delayed)". Should I just wait or should I check the configuration for any errors?

Version 15.0.2

**Comment by caco3** [comment]
you might need to manually refresh the web page.

Also have a look on the log.

If it got delayed on startup it means that it started because of a crash. This sadly is not unusual if something is not configured correctly.
The day is 5 minutes, afterwards it should work as normal.
Instead of waiting you also can reboot the device.

**Reply by lukaszervudakis** [reply]
Just had a random crash. Here is the log:
[log esp.txt](https://github.com/jomjol/AI-on-the-edge-device/files/10844871/log.esp.txt)

I noticed the "Please make sure to setup the SD-Card properly (check the documentation) or re-install using the AI-on-the-edge-device__update__*.zip!"

I formatted a regular micro SD as FAT32 via Windows.

**Comment by lukaszervudakis** [comment, marked as answer]
So I changed the SD card and switched the powersupply. However, I'm still having random reboots and no idea why. Is my board maybe defective?

--> I remade the SD card again with the updated .zip from 15.0.2. Initially I installed an older version and did an OTA update. However, my SD card was still using the files of the older version. It seems to work now, at least when testing at my desk

**Reply by caco3** [reply]
so you ignored the warnings which where shown in the UI?
Please don't do that, the warning is there for a reason!
How ever a wrong UI has no impact in the functionality of the device.

**Reply by lukaszervudakis** [reply]
no. I noticed the warnings and investigated. I initially downloaded 14.0.3 and used its content to copy on the SD card. However, I used the webinstaller to flash the device. Probably the webinstaller was using a more recent version than the SD card files from 14.0.3. After using the SD Card files from 15.0.2 the random reboots stopped.

**Reply by caco3** [reply]
yes, the webinstaller always uses the latest release.
you might just had  done your installation areund the release time.
 
> After using the SD Card files from 15.0.2 the random reboots stopped.

I am glad.
